spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Linear.scala

```diff
@@ -59,28 +54,28 @@ class Linear[T: ClassTag](
 
   val gradWeight: Tensor[T] = Tensor[T]()
   val gradBias: Tensor[T] = if (withBias) Tensor[T]() else null
+
+  private val stdv = 1.0 / math.sqrt(weight.size(2))
+  private var weightInitMethod: InitializationMethod = RandomUniform(stdv)
+  private var biasInitMethod: InitializationMethod = RandomUniform(stdv)
   reset()
 
-  def setInitMethod(initMethod: InitializationMethod): this.type = {
-    this.initMethod = initMethod
+  def setInitMethod(weightInitMethod: InitializationMethod = null,
+                   biasInitMethod: InitializationMethod = null): this.type = {
+    if (weightInitMethod != null) {
+      this.weightInitMethod = weightInitMethod
+    }
+
+    if (biasInitMethod != null) {
+      this.biasInitMethod = biasInitMethod
+    }
+    reset()
     this
   }
 
   override def reset(): Unit = {
```

Comment:
should we merge reset method and setInitMethod. Because it looks like it's more convenient for user. Say if the user wants to change the initMethod:

She/he can write
**linear.reset(xxx, xxx)**
instead of
**linear.setInitMethod(xxx, xxx)**
**linear.reset**